### DISCUSSION TITLE
Need help understanding how the plugin works

### DISCUSSION THREAD

**Discussion by BlitzVI** [natefrisch01/Graph-Link-Types#41] [Q&A]
Hey, so i want to make a plugin of my own which uses the graph view but i've got no help from the discord aside from someone linking me to this plugin. I would love to understand how this plugin and its renderer works so that i can use this knowledge in my plugin aswell.
Any help is much appreciated. Thank you!

**Comment by natefrisch01** [comment]
Well, the first thing I'll say is that the graph view was not designed to be modified. The solution I've used is a hacky one. Essentially we find the renderer that Obsidian makes by waiting a little bit after the graph view opens, and then add things to draw on top of the layer they make. Feel free to use any of my source code as long as you keep the same license and give credit here, and let me know if you have any specific questions about the code.

**Reply by BlitzVI** [reply]
ah do u have a discord or any other way i can contact u directly? i'll be sure to credit you and the plugin if i do use any of the source code and tysm for answering!
